ubuntu/R/README.md

```diff
@@ -1,5 +1,7 @@
 # R container
 
+**Disclaimer** This image is not regularly patched for security updates. It is the user's responsibility to regularly patch and rebuild the images. If concerned, you can always opt to build the containers using your own Dockerfile.
```

Comment:
nit: If concerned, you can always opt to -> If this is a concern, please set automation to regularly rebuild your DCS base images

Reply:
updated both